File: C-Style_to_Python_Syntax_CheatSheet/factorial.py
import factorial_calculator


factCalc = factorial_calculator.FactorialCalculator()

for i in range(1, 11):
    fact = factCalc.calc(i)

    if fact > 100:
        print("Large Answer: {}".format(fact))
    else:
        print("Small Answer: {}".format(fact))

File: C-Style_to_Python_Syntax_CheatSheet/Factorial.java
public class Factorial {
    public static void main(String[] args) {
        FactorialCalculator factCalc = new FactorialCalculator();

        for(int i = 1; i <= 10; ++i) {
            int fact = factCalc.calc(i);

            if(fact > 100) {
                System.out.println("Large Answer: " + fact);
            }
            else {
                System.out.println("Small Answer: " + fact);
            }
        }
    }
}
